data_steward/utils/bq.py

```diff
@@ -240,3 +242,43 @@ def get_latest_validation_dataset_id(project_id):
             return sorted(validation_datasets, key=lambda x: x[0],
                           reverse=True)[0][1]
     return None
+
+
+def create_dataset(dataset_id,
+                   description,
+                   label,
+                   tag,
+                   project_id,
+                   exists_ok=False):
+    """
+    Creates a new dataset
+
+    :param project_id:  name of the project in which to create the dataset
+    :param exists_ok: Defaults to 'False'
+    :param dataset_id:  name to give the new dataset, is required
+    :param description:  dataset description, is required
+    :param label:  dataset label, is required
+    :param tag:  dataset tag, is required
+
+    :raises: RuntimeError if the dataset does not have project_id
+    :raises: RuntimeError if the dataset does not have dataset_id
+    :raises: RuntimeError if the dataset does not have a description
+    :raises: RuntimeError if the dataset does not have a label or tag
+    """
+    if not project_id:
+        raise RuntimeError(
+            "Please specify a project in which to create the dataset")
+
+    if not dataset_id:
+        raise RuntimeError("Cannot create a dataset without a name")
+
+    if not description:
+        raise RuntimeError("Please provide a description to create a dataset")
+
+    if not label or not tag:
+        raise RuntimeError("Label and/or tag is required to create a dataset")
+
+    client = get_client(project_id)
+    client.create_dataset(dataset_id, exists_ok=exists_ok)
```

Comment:
This line creates a dataset with an id.  It doesn't apply any labels, tags, or descriptions.  Have you looked at incorporating the `define_dataset` function?

Reply:
Yes, I will look at the `define_dataset` function, it was suggested to me to follow the `delete_dataset` function at first. I will pivot.

Reply:
Well, deleting is easy.  It may not even need a wrapper function defined here.